raveandroid/src/main/java/com/flutterwave/raveandroid/PayloadEncryptor.java

```diff
@@ -16,7 +14,7 @@ public String getEncryptedData(String unEncryptedString, String encryptionKey) {
 
         if (unEncryptedString != null && encryptionKey != null) {
             try {
-                return encrypt(unEncryptedString, encryptionKey);
+                return Utils.getEncryptedData(unEncryptedString, encryptionKey);
```

Comment:
@jeremiahVaris copy over the function from the `Utils` class into this class. The way we are currently doing it defeats the purpose of this class.

Also, look for all usages of `Utils.getEncryptedData` and replace it with `PayloadEncryptor.getEncryptedData()`. After doing this, do a corresponding change in all the affected tests.

Reply:
This has been done.